Sources/Ink/Internal/CodeBlock.swift

```diff
@@ -13,20 +13,24 @@ internal struct CodeBlock: Fragment {
     private var code: String
 
     static func read(using reader: inout Reader) throws -> CodeBlock {
-        try require(reader.readCount(of: marker) == 3)
-
-        let language = reader.readUntilEndOfLine()
+        let startingMarkerCount = reader.readCount(of: marker)
+        try require(startingMarkerCount >= 3)
+        reader.discardWhitespaces()
+        var language = reader.readUntilEndOfLine()
+        while let l = language.last, l.isWhitespace {
```

Comment:
This could be shortened into `while language.last?.isWhitespace == true`

Reply:
Agreed. I used cleaner form.